### ISSUE TITLE
Misplaced graph x ylabel

### ISSUE THREAD

**Issue by HiIamJeff** [AutoViML/AutoViz#9]
Hi Ram, 
I have tried this package and found out a potential bug. 
When I tried to do the AV.AutoViz('', ',', 'target', df) to run an autoViz stuff, the x y labels of each graph are misplaced (x label should be placed at y label and vice versa.). I have tried two datasets and it still happened. Please look into this and see if this is a bug or I just did something wrong. Thanks!
Jeff

**Comment by morenoh149** [comment]
In future a reproducible colab would be useful to communicate the bug. @HiIamJeff

**Comment by AutoViML** [comment]
Jeff:
Thank you for bringing this to my attention...I'd like to fix it ASAP.
But I agree with Harry. I won't be able to reproduce the problem unless you
host your notebook on your GitHub or host it on a Colab. Hope that's not
too much trouble for you.

**Comment by HiIamJeff** [comment]
Hi @morenoh149 @AutoViML,
Thanks for reminding me. I have put up a Google Colab below. (I don't have much experience with this. Please let me know if you couldn't access) The problem still shows up in this notebook and please check if this is a potential bug. Thanks!
https://drive.google.com/file/d/111P3fRpLrPcS05phMgecqhgStLZDespN/view?usp=sharing

Jeff

**Comment by AutoViML** [comment]
Jeff:
Thanks - you are right => a minor error had crept in when I changed
something recently and it was not detected until now. Eagle eye!
Thanks a lot for bringing it to my attention! I have fixed it in your
notebook and it should look good now.
https://colab.research.google.com/drive/111P3fRpLrPcS05phMgecqhgStLZDespN#scrollTo=n6ORpr-gU9sy


I have updated the "pip install" and the "github" with the correct version.
Thanks again
Ram
